lib/scrabble.rb

```diff
@@ -1,7 +1,31 @@
+require 'pry'
+
 class Scrabble
 
   def score(word)
-    1
+    total_score = 0
+    if word != nil
+      word = word.upcase
+      word = word.split(//)
+      local_hash = point_values
```

Comment:
Get out of this habit. Just access `point_values` by calling it directly. No need to create a `local_hash` to bring it in.